liquid-integration/src/main/java/com/e/liquid_integration/adapter/CryptoCurrencyAdapter.kt

```diff
@@ -0,0 +1,69 @@
+package com.e.liquid_integration.adapter
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.RelativeLayout
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.e.liquid_integration.R
+import com.e.liquid_integration.`interface`.ValueSelectListner
+import com.e.liquid_integration.currency.PayloadItem
+
+
+class CryptoCurrencyAdapter(val _context: Context, val
```

Comment:
Naming variables vith `_` prefix is not common on our project.

Reply:
removed _  before naming variable